tests/grate/asm/fs_vs_tests.txt

```diff
@@ -0,0 +1,13 @@
+alu_buffer_size = 1
+pseq_to_dw_exec_nb = 1
+
+.asm
+EXEC
+	MFU:	var unk(0x104E51BA0) r0.fp20, r0.fp20, r0.fp20, r0.fp20
```

Comment:
It could be so, I haven't looked at it much :)

Isn't the flat-interpolation is what could be done in the lnker? The (dis) modifier https://github.com/grate-driver/grate/wiki/Assembler-syntax#linker-assembler

Maybe noperspective​ interpolation is what could be done within that MFU operation?